tests/integration.rs

```diff
@@ -173,23 +184,59 @@ fn stamp_file() -> TestResult {
 
     // For some reason, we delete the stamp file after `--file` :(
     // Recreate it.
-    run(sweep(&["--stamp"]));
+    run(sweep(&["--stamp"]), target.path());
 
     let actual_cleaned = count_cleaned(&target, args, size)?;
     assert_eq!(actual_cleaned, expected_cleaned);
 
     Ok(())
 }
 
+#[test]
+fn empty_project_output() -> TestResult {
+    let (_size, target) = build("sample-project")?;
+
+    let assert = run(
+        sweep(&["--maxsize", "0"]).current_dir(test_dir().join("sample-project")),
+        target.path(),
+    );
+
+    let output = std::str::from_utf8(&assert.get_output().stdout).unwrap();
+
+    assert_eq!(
+        output,
+        r#"
+[DEBUG] cleaning: "/tmp/.tmpYzB6xH" with remove_older_until_fits
+[DEBUG] size_to_remove: 46978
+[DEBUG] Sizing: "/tmp/.tmpYzB6xH/debug" with total_disk_space_in_a_profile
+[DEBUG] Hashs by time: [
+    (
+        25.123342ms,
+        "261b05c2354104eb",
+    ),
+]
+[DEBUG] cleaning: "/tmp/.tmpYzB6xH/debug" with remove_not_built_with_in_a_profile
+[DEBUG] Successfully removed: "/tmp/.tmpYzB6xH/debug/deps/libsample_project-261b05c2354104eb.rlib"
+[DEBUG] Successfully removed: "/tmp/.tmpYzB6xH/debug/deps/libsample_project-261b05c2354104eb.rmeta"
+[DEBUG] Successfully removed: "/tmp/.tmpYzB6xH/debug/deps/sample_project-261b05c2354104eb.d"
+[DEBUG] Successfully removed: "/tmp/.tmpYzB6xH/debug/.fingerprint/sample-project-261b05c2354104eb"
+[INFO] Cleaned 9.43 KiB from "/tmp/.tmpYzB6xH"
+               "#
+    );
```

Comment:
@jyn514 I inserted it here just to show you, what do you think of matching with RegEx to skip all hashed values?

Reply:
Thanks! That sounds like a good idea :)